packages/core/src/components/ColorPalette/ColorDisplay.tsx

```diff
@@ -133,9 +150,6 @@ export function ColorDisplay({
             <MenuItem onClick={() => onBaseModalOpen()}>
               Edit Base Color
             </MenuItem>
-            <MenuItem onClick={() => onColorNameModalOpen()}>
-              Edit Color Name
-            </MenuItem>
```

Comment:
I think we should keep this way of updating name as well.

Reply:
Agreed, yes let's keep the name